docs/probes.md

```diff
@@ -26,9 +26,6 @@ Make sure to provide enough delay and failureThreshold for the VM and the agent
 
 ### Example
 
-**Note:** The Fedora image used in this example does not have the qemu-guest-agent available by default. We need to install and enable it through the console until this is resolved.
```

Comment:
I don't think you should remove this, we still need to configure cloud-init to install and enable `qemu-guest-agent`.

What about:

**Note:** The Fedora image used in this example does not have `qemu-guest-agent` available by default. We need to install and enable it via cloud-init as shown in the example below:

Reply:
> I don't think you should remove this, we still need to configure cloud-init to install and enable `qemu-guest-agent`.
> 
> What about:
> 
> **Note:** The Fedora image used in this example does not have `qemu-guest-agent` available by default. We need to install and enable it via cloud-init as shown in the example below:

OK, now I get the semantics. I guess you're right. When I read "fedora image" I mentally (chose to) understand "the running VM provided in this example".

Reply:
@marceloamaral would you take another look ?

Reply:
>  Anyway, does it matter what comes on the image ?

I think so, the user could use a custom image that already has `qemu-guest-agent` installed and enabled in systemd.

Reply:
... which would work perfectly with this cloud-init config :) 

Hence my point about the image.

Thanks for the review !